### ISSUE TITLE
BAO not correct displayed

### ISSUE THREAD

**Issue by mkoatwork** [EBISPOT/OLS#147]
Looking at "concentration endpoint" it looks like 
![grafik](https://user-images.githubusercontent.com/19821604/27868902-7b556d1a-619e-11e7-8d82-7ea3181f6a55.png)
Which is different from bioportal 
![grafik](https://user-images.githubusercontent.com/19821604/27868936-9bc5d300-619e-11e7-899e-385168e9fd3f.png)
and even does not make sense to group EC50 under AC50.

**Comment by LLTommy** [comment]
I think you have to simply click on the "Show all siblings" button and your problem is solved, isn't it?

**Comment by mkoatwork** [comment]
The problem is the hierarchy of the terms. EC50 and IC50 are not subsets of AC50. They are separate subclasses. For ECMax the hierarchy is O.K.

Best regards,
Manfred



-----Original Message-----

**Comment by simonjupp** [comment]
In BAO AC50 is defined as being equivalent to EC50 or IC50, in OWL this implies that all EC50 and IC50 are types of AC50.

**Comment by mkoatwork** [comment]
You are right. The description states that AC50 is above IC50 and EC50. But the owl file (http://www.bioassayontology.org/bao/bao_vocabulary_result.owl) states that “EC50” is a subclass of class “concentration response endpoint”

    

    <owl:Class rdf:about="http://www.bioassayontology.org/bao#BAO_0000188">
        <rdfs:subClassOf rdf:resource="http://www.bioassayontology.org/bao#BAO_0002162"/>
        <obo:IAO_0000115>The effective concentration of an agonist, which produces 50% of the maximum possible response for that agonist.</obo:IAO_0000115>
        <rdfs:label>EC50</rdfs:label>
    </owl:Class>

I’ll contact Stefan Schürer to update the ontology accordingly. There are also some other possible bugs in there.

Best regards,
Manfred
